### PR TITLE
Feature sign up

### PR DESCRIPTION
**Pull request by andela-ijubril**

### CODE REVIEW

File: codango/account/views.py [hunk 1]
```diff
@@ -1,6 +1,58 @@
...
+
+    def post(self, request):
+        form = self.form_class(request.POST)
+        if form.is_valid():
+            new_user = form.save()
+            return HttpResponseRedirect('/account/home')
+        else:
+            return render(request, self.template_name, {'form': form})
+#
+#
```
Comment (andela-cnnadi) [inline codango/account/views.py]: Commented out code should not be here.

File: codango/account/templates/account/index.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {% load staticfiles %}
+{% load bootstrap %}
 <!DOCTYPE html>
```
Comment (andela-cnnadi) [inline codango/account/templates/account/index.html]: Re-implement this using base.html

File: codango/account/views.py [hunk 3]
```diff
@@ -1,6 +1,58 @@
...
+#     template_name = 'account/home.html'
+#
+#     def get(self, request):
+#         return render(request, self.template_name)
+#
+#     def post(self, request):
+#         return render(request, self.template_name)
+
+
 def index(request):
```
Comment (andela-cnnadi) [inline codango/account/views.py]: I thought we agreed on Class-based views

File: codango/account/tests/test_views.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase, Client
```
Comment (andela-cnnadi) [inline codango/account/tests/test_views.py]: You did not test your code at all.

**Comment by andela-cnnadi** [comment]
Check the comments and rectify ASAP.

File: codango/account/forms.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+from django import forms 
+from models import UserAccount 
+from django.contrib.auth.models import User
+
+
+class SignUpForm(forms.Form):
+    username = forms.CharField(max_length=300)
+    email = forms.EmailField()
+    password1 = forms.CharField(max_length=100, widget=forms.PasswordInput())
```
Comment (andela-ijubril) [inline codango/account/forms.py]: Need to rename password fields to password and password_conf.

**Pull request closed without merging**